lib/devicetrust/authn/authn.go

```diff
@@ -24,21 +24,25 @@ import (
 	"github.com/gravitational/teleport/lib/devicetrust/native"
 )
 
-// vars below are used to swap native methods for fakes in tests.
-var (
-	getDeviceCredential = native.GetDeviceCredential
-	collectDeviceData   = native.CollectDeviceData
-	signChallenge       = native.SignChallenge
-)
+type CeremonyI interface {
```

Comment:
Let's drop the interface, or at least move the interface to lib/client. (See comments on lib/client.)

We don't want the interface here, it goes against the advice on https://github.com/golang/go/wiki/CodeReviewComments#interfaces.